x-pack/platform/plugins/shared/index_management/public/application/sections/home/index_list/details_page/details_page_mappings_content/mappings_information_panels.tsx

```diff
@@ -87,6 +93,12 @@ export const MappingsInformationPanels = ({
             )}
           </>
         )}
+        <EisCloudConnectPromoCallout
+          promoId="indexDetailsMappings"
+          isSelfManaged={!cloud?.isCloudEnabled}
```

Comment:
A self-managed instance is a cluster that is not cloud and not serverless. Is this code path also not shown for serverless instances? 🤔

Reply:
@sabarasaba  This promo will be shown for self-managed. The `isCloudEnabled` boolean will return true for both ECH and Serverless, as they're both technically cloud. The boolean is set to true if the instance has a `cloudId`.